components/ui/savingIndicator.tsx

```
@@ -0,0 +1,45 @@
+import { Check, Loader2, X } from "lucide-react";
+import { cn } from "@/lib/utils";
+import type { SavingState } from "@/components/hooks/useSavingIndicator";
```

ESLint

[error] Replace `{·cn·}·from·"@/lib/utils";⏎import·type·{·SavingState·}·from·"@/components/hooks/useSavingIndicator` with `type·{·SavingState·}·from·"@/components/hooks/useSavingIndicator";⏎import·{·cn·}·from·"@/lib/utils`

(prettier/prettier)